src/framework/components/element/text-element.js

```diff
@@ -902,7 +904,8 @@ class TextElement {
 
                 var isWordBoundary = WORD_BOUNDARY_CHAR.test(char);
                 var isCJK = CJK_CHAR.test(char);
-                if (isWordBoundary || isCJK) { // char is space, tab, or dash OR CJK
+                var isNextCJK = nextchar ? CJK_CHAR.test(nextchar) : false;
```

Comment:
should this test be `(nextchar !== null) ? ...` ?

Reply:
yes - I'll change it

Reply:
done - thanks!